src/panels/dev-info/ha-panel-dev-info.js

```diff
@@ -292,16 +292,24 @@ class HaPanelDevInfo extends PolymerElement {
     super.connectedCallback();
     this.$.scrollable.dialogElement = this.$.showlog;
     this._fetchData();
+    if (!window.CUSTOM_UI_LIST) {
+      // Give custom UI an opportunity to load.
+      setTimeout(() => {
```

Comment:
This is only necessary if you refresh the page with dev info open right?

Reply:
Yes (or initially open `/dev-info`)